example/android/app/build.gradle

```diff
@@ -47,8 +47,8 @@ android {
         applicationId "com.example.example"
         // You can update the following values to match your application needs.
         // For more information, see: https://docs.flutter.dev/deployment/android#reviewing-the-build-configuration.
-        minSdkVersion flutter.minSdkVersion
-        targetSdkVersion flutter.targetSdkVersion
+        minSdkVersion 28
```

Comment:
Are we sure we need to hardcode here?
@singhtaranjeet any insights?

Reply:
Yes it's fine to have it as hardcoded, totally depends upon the requirements of the project but do we actually need to set it at 28 which is android 9?
Default of flutter 3.16 is minSdkVersion is 19.
And Default of previous flutter version was android 16.

IMO we can go with flutter default as of now but in future if project requires then we can make the values hardcoded.

Reply:
@tajjacob any specific reason we changed this?

Reply:
@apgapg this one for my project. I can change it to default back